packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -15,21 +15,32 @@
  */
 
 import * as types from '@opentelemetry/types';
+import { hrTime } from '@opentelemetry/core';
 import { hashLabelValues } from './Utils';
-import { CounterHandle, GaugeHandle } from './Handle';
+import { CounterHandle, GaugeHandle, BaseHandle } from './Handle';
 import { MetricOptions } from './types';
+import {
+  ReadableMetric,
+  MetricDescriptor,
+  MetricDescriptorType,
+} from './export/types';
 
 /** This is a SDK implementation of {@link Metric} interface. */
 export abstract class Metric<T> implements types.Metric<T> {
   protected readonly _monotonic: boolean;
   protected readonly _disabled: boolean;
   protected readonly _logger: types.Logger;
+  private readonly _metricDescriptor: MetricDescriptor;
   private readonly _handles: Map<String, T> = new Map();
```

Comment:
curious why `String` instead of `string` ?

Reply:
Changed in https://github.com/open-telemetry/opentelemetry-js/pull/437/commits/dde4d20ffe872059a86e14d045293c18b01a76e8